blueprints/addon/index.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     contents.devDependencies['ember-disable-prototype-extensions'] = '^1.1.0';
 
     // add `ember-try` to addons by default
-    contents.devDependencies['ember-try'] = '~0.0.8';
+    contents.devDependencies['ember-try'] = '^0.1.2';
```

Comment:
^ will not change the first non-zero number. 

^0.1.2 will allow everything below 0.2.0, ^1.0.0 will allow everything below 2.0.0

---

Also, 0.1.0 wasn't a breaking change, the older syntax for config/ember-try.js still works with 0.1.0 but using that syntax doesn't allow controlling npm deps.